### ISSUE TITLE
Name property is not deﬁned for the list items under Azure section in Linked accounts dialog:A11y_SQL Azure Data Studio_Azure Tree item_AI4D

### ISSUE THREAD

**Issue by sspatil123** [microsoft/azuredatastudio#26036]
Please do not close this bug. This bug should only be closed by Trusted Tester after veriﬁcation 
"Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug ﬁxing faster and easier." 

### GitHub Tags:
#A11yWCAG; #A11ySev2; #A11yTCS; #DesktopApp; #BM_SQL Azure Data Studio_Win32_Nov2024; #SQL Azure Data Studio; #WCAG4.1.2;  #Win32; #Benchmark; #FTP; #AI4D;#NameRoleValue;

### Environment Details:
Application Name: SǪL Azure Data Studio 
Edition: Windows 11 Enterprise 
Version: 24H2 
Installed: 22-10-2024 
OS build: 26100.2033 
Experience: Windows Feature Experience Pack 1000.26100.23.0 

### Steps to Reproduce
1.Launch the application-Azure Database Project and install extension SǪL Database Projects. 
2.Tab to “Connections” tab item present in the left menu and Hit Enter Key  
3.Tab to “New connection” button in the connections section and Hit ENTER key. 
4.Tab to “Browse” tab and Hit ENTER key. 
5.Tab to “+” (Azure: sign in) button in the Azure tree item. 
6.“Linked Accounts” dialog will get open. 
7.Tab to “list items” under Azure section. 
8. Open accessibility insights for windows and check whether name property is deﬁned for the list items or not. 

### Actual Results:
Name property is not deﬁned for the list items under Azure section in Linked accounts dialog. 

### Expected Results:
Name property should be deﬁned for the list items under Azure section in Linked accounts dialog. 

### 
### User Impact:
Screen reader users may face difficulty while navigating to the controls if the screen reader will not announce the name for the controls. 

### Attachment:
![Image](https://github.com/user-attachments/assets/e5810a70-dc7c-48ae-82d5-25d861605252)
https://github.com/user-attachments/assets/ac7b6cf3-36d1-425d-8a7c-55c8494640bf
